galleon-pack/galleon-feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -86,7 +85,6 @@
                 <extension>org.wildfly.extension.microprofile.fault-tolerance-smallrye</extension>
                 <extension>org.wildfly.extension.microprofile.jwt-smallrye</extension>
                 <extension>org.wildfly.extension.microprofile.openapi-smallrye</extension>
-                <extension>org.wildfly.extension.microprofile.opentracing-smallrye</extension>
```

Comment:
I think this should stay. The extension still exists so Galleon should be aware of it.

Reply:
Fixed